domains/plsb.json

```diff
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "arizkinewbie",
+    "email": "me@arizkinewbie.com"
+  },
+  "records": {
+    "NS": [
+      "ns1.domainesia.net",
+      "ns2.domainesia.net"
+    ]
```

Comment:
Denied as no valid reasoning was provided on why NS records are needed.

Reply:
the reason I use nameservers is so that it can be managed directly on the server with DNS zones Sir